bom/pom.xml

```diff
@@ -4,55 +4,55 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.openhab.addons.bundles</groupId>
+    <groupId>org.openhab.addons.bom</groupId>
```

Comment:
It should reference the GAV parameters of the parent POM. With these changes it refers to a parent POM that doesn't exist.

Instead you could override the groupId tag from the parent by adding a new `<groupId>org.openhab.addons.bom</groupId>` 
above the
`<artifactId>org.openhab.addons.bom.zigbee</artifactId>`
below. :arrow_down: